mobile/src/home-view.js

```diff
@@ -354,10 +354,16 @@ class HomeView extends PureComponent {
   }
 
   addCard = newCard => {
-    const cards = [...this.state.cards, newCard]
-    this.cardsRef().set(cards)
-    this.saveLocalCards({ myCard: this.state.myCard, cards })
-    this.setState({ cards, showScanner: false })
+    if (newCard.firstName && newCard.email) {
+      const cards = [...this.state.cards, newCard]
+      this.cardsRef().set(cards)
+      this.saveLocalCards({ myCard: this.state.myCard, cards })
+      this.setState({ cards, showScanner: false })
+    } else {
+      Alert.alert(t('error'), t('newScan'), [{ text: 'OK', onPress: () => console.log('OK') }], {
```

Comment:
is the console.log necessary?  Also, isn't `OK` the default button if none are specified?